src/js/helpers/svg.js

```diff
@@ -1,7 +1,9 @@
+window.baseUrl = '/Frontend-build-sandbox/build';
```

Comment:
It's global change for everyone. It's ready for use with http://localhost:3333/header-user.html and httpster https://github.com/jacekpolak/httpster 

On master there is good version.